.travis.yml

```diff
@@ -11,6 +15,14 @@ branches:
 
 install:
   - npm ci
+  - git clone https://github.com/DefinitelyTyped/DefinitelyTyped ../DefinitelyTyped
+
+script:
+  - npm tes
+  - npm run build
+  - npm run clean
+  - npm run parse
+  - npm run check
```

Comment:
1. This isn't very valuable, it mostly just tells you what `@types` packages are superceded by in-package types.
2. On the other hand, we shouldn't *never* run it.
3. On the third hand, it'll make CI slow.
4. Is CI never not slow?
3. This is probably the right place to run it.